src/Console/Shell.php

```diff
@@ -211,7 +211,10 @@ public function initialize()
      */
     public function startup()
     {
-        $this->_welcome();
+        $requested = $this->param('requested');
+        if (empty($requested)) {
```

Comment:
Couldn't you just test the truthiness of the param? Given that its always locked to a boolean it will be a yes/no condition.